src/test_runner.rs

```diff
@@ -16,6 +19,8 @@ use osmosis_test_tube::OsmosisTestApp;
 /// An enum with concrete types implementing the Runner trait. We specify these here because the
 /// Runner trait is not object safe, and we want to be able to run tests against different types of
 /// runners.
+#[derive(strum::EnumVariantNames)]
+#[strum(serialize_all = "kebab_case")]
```

Comment:
what does this do?

Reply:
generates a constant `TestRunner::VARIANTS` that can be used in the tests. If we add a new variant without also updating the relevant tests they will fail